Sources/ZcashLightClientKit/Tool/DerivationTool.swift

```diff
@@ -166,7 +171,7 @@ public class DerivationTool: KeyDeriving {
     
     - Returns: the spending keys that correspond to the seed, formatted as Strings.
     */
-    public func deriveSpendingKeys(seed: [UInt8], numberOfAccounts: Int) throws -> [String] {
+    public func deriveSpendingKeys(seed: [UInt8], numberOfAccounts: Int) throws -> [SaplingExtendedSpendingKey] {
```

Comment:
```suggestion
    public func deriveSaplingExtendedSpendingKeys(seed: [UInt8], numberOfAccounts: Int) throws -> [SaplingExtendedSpendingKey] {
```
(Modulo return-arity issue)